Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -934,9 +932,32 @@ bool Renderer::StartFrameDumpToImage(const FrameDumpConfig& config)
   return true;
 }
 
+void writePPM(std::ostream& os, int width, int height, int stride, const u8* data)
+{
+  using namespace std;
```

Comment:
From the contributing guide:
>Do not use using namespace [x]; in headers. Try not to use it at all if you can.

Its use can definitely be avoided here, especially for the sake of consistency.